app/react/Viewer/PDFView.js

```diff
@@ -53,6 +54,10 @@ class PDFView extends Component {
     }
   }
 
+  componentWillUnmount() {
+    this.context.store.dispatch(leaveEditMode());
```

Comment:
I think we should connect PDFView instead of using actions this way.

Reply:
It's done. I had some problems at testing but decided to update just for the changed part.